mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,35 @@
+// This custom encoder helps prevent sending 0 for clientTtl, defaultTtl and
+// maxTtl in PATCH calls when unset in the provider (doing so results in an API
+// level error)
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+// Only apply during update
+if !cdnPolicyOk {
+	return obj, nil
+}
+
+// Don't do anything if we're not PATCHing
+if obj["cdnPolicy"] == nil {
+	return obj, nil
+}
+
+cdnPolicies, err := c.([]interface{})
```

Comment:
@ScottSuarez This was my attempt at resolving this error with applying using the built provider:

```
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-google plugin:

panic: interface conversion: interface {} is nil, not string
```

so I'm thinking if this interface is nil, should be fine to return `obj, nil` there?